src/components/tab/WorkspaceTab.vue

```diff
@@ -0,0 +1,93 @@
+
+<template>
+<el-card class="box-card">
+        <div slot="header" class="clearfix">
+            <slot name="title">Workspace</slot>
+        </div>
+        <slot name="items">
+            <el-tabs v-model="activeName" :title="title" :tab-position="tabPosition" @tab-click="$emit('click')">
+            <slot></slot>
+            </el-tabs>
+        </slot>
+    </el-card>
+</template>
+
+<script>
+// import { action } from '@storybook/addon-actions'
+
+export default {
+  name: 'WorkspaceTab',
+  props: {
+    title: {
+      type: String,
+      default: ''
+    },
+    tabPosition: {
+      type: String,
+      default: ''
+    },
+    activeName: {
+      type: String,
+      default: ''
+    }
+  },
+  methods: {
+    click (tab, event) {
+      console.log(tab, event)
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+@import "../../styles/theme";
+
+// tab styling
+.el-tabs_item {
+    font-family: $main-font-family;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 22px;
+    color: $dark-body-grey;
+    text-align: left;
```

Comment:
I found this other class that is aligning the text to left. you can check it out
```
.el-tabs--left .el-tabs__item.is-left{
  text-align: left;
}
```